onnxruntime_extensions/_hf_cvt.py

```diff
@@ -24,16 +25,31 @@ def __init__(self, tokenizer):
     def bpe_tokenizer(self, **kwargs):
         hf_gpt2_tokenizer = self.tokenizer
 
-        if ("Fast" in str(self.tokenizer)):
+        if ("PreTrainedTokenizerFast" in str(hf_gpt2_tokenizer)):
+            tokenizer_json_path = f"https://huggingface.co/{hf_gpt2_tokenizer.init_kwargs.get('name_or_path')}/raw/main/tokenizer.json"
+            tokenizer_json = json.loads(requests.get(tokenizer_json_path).text)
+            
+            attrs = {}
+            attrs['vocab'] = f'{tokenizer_json["model"]["vocab"]}'
+            sorted_merges = {}
+            i = 0
+            for merge in tokenizer_json["model"]["merges"]:
+                sorted_merges[i] = (merge[:merge.index(" ")], merge[merge.index(" ")+1:])
+                i += 1
+            attrs['merges'] = '\n'.join("{} {}".format(
+                *sorted_merges[n_]) for n_ in range(len(sorted_merges)))
+            attrs.update(**kwargs)
```

Comment:
parsing of vocab and merges need to be fixed - `PreTrainedTokenizerFast` object has vocab but not merges, so they need to be parsed in from somewhere - I have this implementation right now, but it's broken.

Reply:
it doesn't work. Are you always assuming all vocab and merges are same?